shell/mac/ViewController.mm

```diff
@@ -196,6 +196,10 @@ - (void)loadView {
 
 #if IGL_BACKEND_OPENGL
   case igl::BackendFlavor::OpenGL: {
+    const bool enableStencilBuffer = config_.depthTextureFormat == igl::TextureFormat::S8_UInt_Z24_UNorm  ||
```

Comment:
If you extract all these changes in `shell/mac/ViewController.mm` into a separate PR, it will be easier to review and merge because it does not change any existing behaviors.